apps/chronicles/screens/components/components/ProgressScreen.tsx

```diff
@@ -0,0 +1,157 @@
+    const { tasks, startUploadSimulation, handleCopyErrorMessage, handleRetry } =
+        useProgressUpload();
+    const styles = useStyles(themeStyles);
+
+    const handleUploadSuccess = async () => {
+        await startUploadSimulation("success");
+    };
+
+    const handleUploadFailed = async () => {
+        await startUploadSimulation("fail");
+    };
+
+    return (
+        <ScrollView
+            contentInsetAdjustmentBehavior="automatic"
+            contentContainerStyle={styles.contentContainer}
+        >
+            <Typography group="basic" variant="h2">
+                Progress
+            </Typography>
+
+            <Typography>
+                The Progress component can be used for tracking and displaying the progress of tasks
+                or processes, such as «create folder» or «upload images».
+            </Typography>
+            <Typography>
+                Progress can be used with one or multiple Progress Items, and one Progress Item can
+                contain one single task or multiple tasks.
+            </Typography>
+
+            <Spacer amount="small" />
+            <Typography>
+                Tasks have different statuses that is based on the overall progress.
+            </Typography>
+            <Typography> Status can either be:</Typography>
+            <View style={{ flexDirection: "row", gap: 8, alignItems: "center" }}>
+                <Typography color="primary">inProgress</Typography>
+                <CircularProgress size={18} value={0.7} />
+                <Typography color="textTertiary">, notStarted, </Typography>
+                <Typography color="success">sucess</Typography>
+                <Typography>or</Typography>
+                <Typography color="danger">error</Typography>
+            </View>
+            <Spacer amount="small" />
+            <UploadSimulator
+                onUploadSuccess={() => void handleUploadSuccess()}
+                onUploadFailed={() => void handleUploadFailed()}
+            />
+
+            <Spacer />
+
+            <Typography>Progress with one single task: </Typography>
+
+            <Spacer amount="small" />
+            <Progress title="Create folder">
+                <Progress.Item
+                    title="Creating cat images folder"
+                    description="This folder contains cat images"
+                    status="success"
+                />
+            </Progress>
+
+            <Spacer />
+
+            <Typography>Progress with multiple tasks: </Typography>
+            <Spacer amount="small" />
+            <Progress title="Upload cat images">
+                <Progress.Item
+                    title="Upload images of cats with hats"
+                    description="uploading cats with hats"
+                    tasks={tasks}
+                    showCopyTextButton
+                    showRetryButton
+                    onCopyTextButtonPress={handleCopyErrorMessage}
```

Comment:
Should we maybe just show the copy button if the `onCopyTextButton` prop is provided? This means there will be less props for the user to use so that they don't have to add both props (one for showing, and one for the onPress).

Same goes for the retry button